docker/stand-alone/docker-compose.yaml

```diff
@@ -23,7 +23,7 @@ services:
       - sam-app:app
       - sam-opendj:opendj
   sam-opendj:
-    image: broadinstitute/openam:opendj5
```

Comment:
No clue what this was… a different tag? We should probably just use all the same tag

Reply:
I'm not familiar with this file/directory at all, maybe @gpolumbo-broad can weigh in as a past contributor to the file? Is it used for anything? I wouldn't mind removing it entirely if it's not used anymore (not in this PR but as a future tech debt item)

Reply:
Oh, the old stand-alone stuff was from way back when I first started and there was a request to set up stand-alone sam.  I don't remember why or if it was ever used.  I suspect that we can probably remove it now, but I'm not sure.